ws-services/src/main/java/org/egov/waterconnection/util/WaterServicesUtil.java

```diff
@@ -99,6 +100,21 @@ public AuditDetails getAuditDetails(String by, Boolean isCreate) {
 				&& "EMPLOYEE".equalsIgnoreCase(waterConnectionRequest.getRequestInfo().getUserInfo().getType())) {
 			propertyCriteria.setTenantId(waterConnectionRequest.getWaterConnection().getTenantId());
 		}
+		if (waterConnectionRequest.getRequestInfo().getUserInfo() != null
+				&& "SYSTEM".equalsIgnoreCase(waterConnectionRequest.getRequestInfo().getUserInfo().getType())) {
```

Comment:
What is this code for?

Reply:
Here we are handling the open public search for property and adding the required search criteria and added the requied role for it.